scripts/setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+echo "Configuring System for Distributed Bitmap Engine"
+
+UPDATEFILE="/home/ubuntu/.dbe.checked-updates"
+DEPENDFILE="/home/ubuntu/.dbe.checked-dependencies"
```

Comment:
why not just use `~` instead of `/home/ubuntu` for portability

Reply:
Done.